libs/signed_integers/src/i128.sw

```diff
@@ -5,15 +5,35 @@ use ::common::TwosComplement;
 use ::errors::Error;
 
 /// The 128-bit signed integer type.
+///
+/// # Additional Information
+///
 /// Represented as an underlying U128 value.
 /// Actual value is underlying value minus 2 ^ 127
 /// Max value is 2 ^ 127 - 1, min value is - 2 ^ 127
 pub struct I128 {
+    /// The underlying unsigned number representing the `I128` type.
     underlying: U128,
 }
 
 impl I128 {
-    /// The underlying value that corresponds to zero signed value
+    /// The underlying value that corresponds to zero signed value.
+    ///
+    /// # Returns
+    ///
+    /// * [U128] - The unsigned integer value representing a zero signed value.
```

Comment:
No matter how many times I read these two sentences I cannot make sense of them

Reply:
It's the zero value. Not sure what there is not to understand?